app/services/tree-component-service.ts

```diff
@@ -0,0 +1,14 @@
+import { IActionMapping } from "angular-tree-component";
```

Comment:
I don't think this makes sense to have it as a service. There techincally could be multiple tree view showing different tree at the same time